README.md

```diff
@@ -2,9 +2,95 @@
 
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-MicroCeph is snap-deployed Ceph with built-in clustering.
+<p align="center">
+[MicroCeph](https://snapcraft.io/microceph) is snap-deployed Ceph with built-in clustering.
+</p>
 
-MicroCeph handles scaling out the monitor cluster, as well as automating placement and management of the manager and metadata processes in a Ceph cluster when units are dynamically added via easy-to-use tooling. It also helps with adding disks (OSDs) to the Ceph cluster.
+## 💡 Philosophy
 
-Documentation: https://canonical-microceph.readthedocs-hosted.com/en/latest/
-Snap: https://snapcraft.io/microceph
+Deploying and operating a Ceph Cluster is complex, It is because Ceph is designed to be a general purpose Storage solution. This is a significant overhead for smaller Ceph clusters. [MicroCeph](https://snapcraft.io/microceph) solves this by being Opinionated and Focused at small scale. With Microceph, deploying and operating a Ceph cluster is as easy as a [Snap!](https://snapcraft.io/microceph)
+
+## 🎯 Features
+
+1. Quick and Consistent Deployment with minimal overhead.
+2. Single-command operations (for bootstrapping, adding OSDs, service enablement etc).
+3. Isolated from Host and Upgrade friendly.
+4. Built-in clustering so you don't have to worry about it!
+5. Tailored for small scale (Or Just your Laptop).
+
+## 📖 Documentation
+
+Refer [QuickStart](#⚡️-quickstart) for quick help. If you want to read Official Documentation, please visit our hosted [Documentation](https://canonical-microceph.readthedocs-hosted.com/en/latest/).
+
+## ⚡️ Quickstart
```

Comment:
I wonder if it might be worth splitting quickstart into two sections vertically so that we could have something like ceph dashboard screeenshots on the right side with the code / text along the left

Reply:
Hmm, I feel it may appear distracting, however, let me try setting it up that way once.